share/github-backup-utils/ghe-backup-config

```diff
@@ -66,6 +66,39 @@ for f in "$GHE_BACKUP_CONFIG" "$GHE_BACKUP_ROOT/backup.config" \
   fi
 done
 
+# Check that moreutils ts has been installed
+if ! echo "moreutils" | ts &>/dev/null; then
+  echo "Error: moreutils not found. Please install https://joeyh.name/code/moreutils" 1>&2
+  exit 1
+fi
+
+ghe_parallel_check() {
+  # Some machines may have both moreutils parallel and GNU parallel installed.
+  GHE_PARALLEL_COMMAND="parallel"
+  if [ -x "/usr/bin/parallel.moreutils" ]; then
+    GHE_PARALLEL_COMMAND="/usr/bin/parallel.moreutils"
+  fi
+
+  # Check that the GHE_PARALLEL_COMMAND is pointing to moreutils parallel
+  if ! $GHE_PARALLEL_COMMAND -h | grep -q "parallel \[OPTIONS\] command -- arguments"; then
+    echo "Error: moreutils not found. Please install https://joeyh.name/code/moreutils" 1>&2
+    exit 1
+  fi
+
+  GHE_PARALLEL_COMMAND_OPTIONS=""
+  if [ "$GHE_PARALLEL_ENABLED" != "yes" ]; then
```

Comment:
This was a good idea to put it behind configuration. I assume there's a commit that just hasn't been pushed up that includes this and some comments in the `backup.config-example` file ?

Reply:
I think @itillis doesn't want us to advertise this flag in our documentation just yet.

Reply:
I've added this flag to our documentation and marked it as a beta feature.